Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -168,6 +171,20 @@ extension _EasyHandle {
         let protocols = (CFURLSessionProtocolHTTP | CFURLSessionProtocolHTTPS)
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
+        #if os(Android)
+            // See https://curl.haxx.se/docs/sslcerts.html
+            // For SSL to work you need "cacert.pem" to be accessable
+            // at the path pointed to by the URLSessionCAInfo env var.
```

Comment:
If you want to follow the naming conventions put forth for Foundation API; 

1) if you have a setter you should have a getter (unless it is absolutely justifiable that it can only be set)
2) it should be a URL not a path
3) don't use abbreviations like CA, instead it should be probably named `URLSession.defaultCertificateAuthorityFile` or something along those lines so that it does what it says on the tin.
4) it should probably validate that the url is a valid file path url, and perhaps it should be nullable?
5) it should probably also be atomic (using some sort of lock to ensure thread safe access)